portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -35,53 +35,56 @@
 import com.google.sps.data.Comment;
 
 /** 
-* Servlet that creates comment entities and redirects the user back to the 
-* blog section of the portfolio page.
-*
-* @param request which contains data to retrieve comments
-* @param response
-* @return comments in the form of json
-*/
+ * Servlet that creates comment entities and redirects the user back to the 
+ * blog section of the portfolio page.
+ *
+ * @param request which contains data to retrieve comments
+ * @param response
+ * @return comments in the form of json
+ */
 @WebServlet("/list-comments")
 public class ListCommentsServlet extends HttpServlet {
 
-  /** Will only show the 30 most recent comments.
-      Returns a List<Comment> */
+  static final int MAX_NUM_COMMENTS = 5;
+  static final int MAX_NUM_BLOGS = 5;
```

Comment:
Since this is repeated in several places, can you create a class to contain constants like this? That way, you'll only have to specify this in one place.